Allow "OT" as a two-letter code for the project. (#4003)
open-telemetry/opentelemetry-specification: specification/README.md

- The official project name is "OpenTelemetry" (with no space between "Open" and
  "Telemetry").
- The official acronym used by the OpenTelemetry project is "OTel". "OT" MAY be

>   used as a two-letter acronym where appropriate, such as in a multi-word
>   phrase (e.g., "OpenTelemetry Certified Associate").

Reviewer: ```suggestion
  used only as a part of a longer acronym, such as OTCA (OpenTelemetry Certified Associate).
```
